emission/core/wrapper/common_place.py

```diff
@@ -0,0 +1,17 @@
+import logging
+import emission.core.wrapper.wrapperbase as ecwb
+
+class CommonPlace(ecwb.WrapperBase):
+    props = {"coords" : ecwb.WrapperBase.Access.WORM, # geojson coordinates that represent the place
+             "edges" : ecwb.WrapperBase.Access.WORM, # set of CommonTrips that include this location
+             "successors" : ecwb.WrapperBase.Access.WORM, # set of CommonPlaces connected to this location
```

Comment:
Just so that I understand, we don't technically need the successors, right? We can just iterate through the edges and find the destination node for each of them. So this is a convenience function that is duplicated. Correct?